lib/flipper/api/v1/actions/boolean_gate.rb

```diff
@@ -0,0 +1,21 @@
+require 'flipper/api/action'
+
+module Flipper
+  module Api
+    module V1
+      module Actions
+        class BooleanGate < Api::Action
+          route %r{api/v1/features/[^/]*/(enable|disable)/?\Z}
+          
+          def put
+            feature_name = Rack::Utils.unescape(request.path.split("/")[-2])
+            feature = flipper[feature_name.to_sym]
+            action = Rack::Utils.unescape(request.path.split("/").last)
+            feature.send(action)
```

Comment:
Let's verify that the action is one in a valid set of actions.

```ruby
VALID_ACTIONS = Set.new(["enable", "disable"]).freeze

if VALID_ACTIONS.include?(action)
  feature.send(action)
  json_response ...
else
  ... some response that says it is not valid action
end
```

Or:

case action
when "enable"
  feature.enable
when "disable"
  feature.disable
else
  ... some json response that says action is not valid
end
```

I don't want someone to be able to call a ruby method based on the url.